### ISSUE TITLE
App Fails to Start After Restoring Internet Connection

### ISSUE THREAD

**Issue by cahlenlee** [lbryio/lbry-android#1070]
**DESCRIPTION:**

If there is no internet connection on the first launch of the app the user will receive an error telling them to check their connection and try again. If the user then restores the internet connection and tries again they will receive the same error. 

-----

**REPRO STEPS:**
1) Disconnect phone from internet
2) Install and Launch LBRY 0.16.11 on Android 8.1.0
3) Observe error message
4) Reconnect phone to internet
5) Press Home button
6) Launch LBRY again

**Results:** Error message

**Expected:** The app starts

-----

**ENVIRONMENT:**

- LG V20
- Android 8.1.0
- LBRY 0.16.11

-----

**MEDIA:**

![LBRY_16 11_InitialStartWiFi](https://user-images.githubusercontent.com/30788194/102130355-fde3e500-3e0d-11eb-8d78-99a8ec1aa983.gif)

-----

**LBRY ADDRESS:**
bZSqrVdpxt5KJdn2eGMUzT9Jn3gEiRet7b
mail@cahlen.org

**Comment by tzarebczan** [comment]
Did you kill the app from the notification area also? That would probably need to be done. We'd have to update the text to say that.

**Comment by cahlenlee** [comment]
No, just clicked the home button. Can't the app just restart the daemon when you return?
